src/client/pages/Landing/components/Card.tsx

```diff
@@ -80,6 +80,7 @@ const CardHeader = styled.div`
   }
 
   @media (min-width: 1020px) {
+    flex-direction: row-reverse;
     padding-bottom: 3rem;
     svg {
       display: block;
```

Comment:
This works fine I think 🙌. Alternatively we can just switch the order of `<Badge>` and `<HedvigSymbol>` in the component and set `margin-left:auto` on the svg, if that feels simpler 🙃
```suggestion
  @media (min-width: 1020px) {
    padding-bottom: 3rem;
    svg {
      display: block;
      margin-left: auto;
```

Reply:
I just want to make it clear - it's not obvious what "row-reverse" does here :P

Will sleep on it - thanks for the suggestion!